config/sidekiq.yml

```diff
@@ -3,8 +3,8 @@
 :concurrency:  2
 :logfile: ./log/sidekiq.json.log
 :queues:
-  - content_store_high # Deprecated, to be removed once jobs completed
-  - content_store_low # Deprecated, to be removed once jobs completed
+  - content_store_high
```

Comment:
What's the reason for keeping separate queues for these now? I think we should move all content store/downstream jobs to `downstream_*`